src/Integration/Monolog/CloudWatch/tests/CloudWatchLogsHandlerTest.php

```diff
@@ -44,7 +44,7 @@ public function testConfig()
             'stream' => $this->streamName,
         ], Logger::CRITICAL, false);
 
-        self::assertEquals(Logger::CRITICAL, $handler->getLevel());
+        self::assertEquals(Logger::CRITICAL, $handler->getLevel()->value);
```

Comment:
This won't work for tests runing on lower version of PHP.
You should add something like 
```php
if (enum_exists(Level)) {
  self::assertEquals(Logger::CRITICAL, $handler->getLevel()->value);
} else {
  self::assertEquals(Logger::CRITICAL, $handler->getLevel());
}
```

Reply:
rolled back